uint16 blockid
uint16 meta
float64 x
float64 y
float64 z
